tests/containers/buildx.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: buildx
+# Summary: test buildx plugin
+# - install buildx
+# - build test image
+# - run container with test image
+# - cleanup
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use utils;
+use version_utils qw(is_transactional);
+use containers::common;
+
+sub run {
+    select_serial_terminal;
+
+    my $docker_info = script_output("docker info");
+    record_info('Docker info pre-install', $docker_info);
+
+    my $pkgs = 'docker-buildx';
+    if (is_transactional) {
+        trup_call("pkg install $pkgs");
+        check_reboot_changes;
+    } else {
+        zypper_call("in $pkgs");
+    }
+
+    $docker_info = script_output("docker info");
+    record_info('Docker info post-install', $docker_info);
+
+    die if ($docker_info !~ /plugins\/docker-buildx/);
+
+    my $test_image = "test_buildx";
+    my $test_container = "test_buildx";
+    assert_script_run("echo -e 'FROM busybox\nEXPOSE 5000' > Dockerfile");
+    assert_script_run("docker build -t $test_image .");
+    assert_script_run("docker run --rm -d -p 54321:5000 --name $test_container $test_image");
+    assert_script_run("docker stop $test_container");
+    assert_script_run("docker rm -v $test_container");
```

Comment:
This is without effect, as there is `--rm` in line 44 and all calls are `assert_script_run`. I would simply remove it or do a proper cleanup routine in the `post_{run,fail}_hooks`